### PR TITLE
GCM -> FCM conversion

### PR DESCRIPTION
**Pull request by spelluru**
@jwargo - I have tested and updated the Xamarin Android tutorial. Now, checking in it as a sample here. Please review when you get a chance. Thanks.

### CODE REVIEW

File: Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/Assets/AboutAssets.txt [hunk 1]
```diff
@@ -1,19 +1,19 @@
 Any raw assets you want to be deployed with your application can be placed in
 this directory (and child directories) and given a Build Action of "AndroidAsset".
 
-These files will be deployed with your package and will be accessible using Android's
+These files will be deployed with you package and will be accessible using Android's
```
Comment (mpodwysocki) [inline Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/Assets/AboutAssets.txt]: Why was this changed? Having "your" should be correct here.

File: Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/Constants.cs [hunk 2]
```diff
@@ -1,13 +1,20 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
```
Comment (mpodwysocki) [inline Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/Constants.cs]: Remove and sort `using` statements.  None of these should be necessary.

File: Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/MyFirebaseIIDService.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
```
Comment (mpodwysocki) [inline Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/MyFirebaseIIDService.cs]: Remove/sort `using` statements as above.  Most of these should not be required

File: Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/MyFirebaseMessagingService.cs [hunk 4]
```diff
@@ -0,0 +1,68 @@
+﻿using System;
```
Comment (mpodwysocki) [inline Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/MyFirebaseMessagingService.cs]: Same as above, these `using` statements should be sorted and those not used should be removed.

File: Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/MainActivity.cs [hunk 5]
```diff
@@ -1,55 +1,80 @@
...
-			Log.Info("MainActivity", "Registering...");
-			GcmClient.Register(this, Constants.SenderID);
-		}
-	}
-}
+    [Activity(Label = "@string/app_name", Theme = "@style/AppTheme", MainLauncher = true)]
+    public class MainActivity : AppCompatActivity
+    {
+        public const string TAG = "MainActivity";
+        internal static readonly string CHANNEL_ID = "my_notification_channel";
```
Comment (mpodwysocki) [inline Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/MainActivity.cs]: The `CHANNEL_ID` should be renamed to `ChannelId` to conform to .NET Naming standards
Why is this internal static readonly instead of a constant?  
Should this be in the constants file because we make no mention of it in our docs down below?

File: Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/Assets/AboutAssets.txt (hunk 1 above)
Comment (spelluru) [reply Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/Assets/AboutAssets.txt]: Don't remember changing it. Fixing it now. Thanks.

File: Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/Constants.cs (hunk 2 above)
Comment (spelluru) [reply Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/Constants.cs]: Fixing it now.

File: Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/MainActivity.cs (hunk 5 above)
Comment (spelluru) [reply Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/MainActivity.cs]: Yes. Making it a constant.
